micro-rdk/src/common/ota.rs

```diff
@@ -49,15 +49,18 @@ use crate::esp32::esp_idf_svc::{
     ota::{EspFirmwareInfoLoader, EspOta},
     sys::EspError,
 };
+use async_io::Timer;
 use http_body_util::{BodyExt, Empty};
 use hyper::{body::Bytes, client::conn::http2, Request};
 use once_cell::sync::Lazy;
+use std::time::Duration;
 use thiserror::Error;
 #[cfg(not(feature = "esp32"))]
 use {bincode::Decode, futures_lite::AsyncWriteExt};
 
 // TODO(RSDK-9200): set according to active partition scheme
 const OTA_MAX_IMAGE_SIZE: usize = 1024 * 1024 * 4; // 4MB
+const CONN_RETRY_SECS: u64 = 60;
```

Comment:
Rather than the constant being a u64 with `SECS` in the name, and then constructing a Duration from it each time, could this constant just **be** the Duration?

Reply:
yeah, i did that initially as a local variable to reuse in logs, but `Duration` doesn't impl display. though now that I think about it, Duration does have an `as_secs` method that returns a `u64`, so I'll use that.